engine/admin/volumes/volumes.md

```diff
@@ -97,7 +97,7 @@ container.
 **Create a volume**:
 
 ```bash
-$ docker volume create my-vol
+$ docker volume create --name my-vol
```

Comment:
hm, not this isn't correct.

are you perhaps running an old version of docker? the `--name` flag is now hidden, and only kept for backward compatibility; the new format is to use the name as is documented here